cmake/modules/FindHostTools.cmake

```diff
@@ -48,7 +48,7 @@ if(HostTools_FOUND)
   return()
 endif()
 
-find_package(Deprecated COMPONENTS XCC_USE_CLANG CROSS_COMPILE)
+find_package(Deprecated COMPONENTS XCC_USE_CLANG CROSS_COMPILE XTOOLS)
```

Comment:
This should be done here:
https://github.com/zephyrproject-rtos/zephyr/blob/4baf2eb75150a11b59c57670ddbbfe7c7e9c36d2/cmake/toolchain/xtools/generic.cmake#L2

as:
```
find_package(Deprecated COMPONENTS XTOOLS)
```

that way we already know user has specified `xtools`.